alert-database/src/main/resources/liquibase/6.4.0/static-distribution-jobs.xml

```diff
@@ -0,0 +1,399 @@
+        <addForeignKeyConstraint
+                constraintName="email_job_additional_email_addresses_job_id_fk"
+                baseTableSchemaName="alert"
+                baseTableName="email_job_additional_email_addresses"
+                baseColumnNames="job_id"
+                referencedTableSchemaName="alert"
+                referencedTableName="distribution_jobs"
+                referencedColumnNames="job_id"
+                onDelete="CASCADE"
+                onUpdate="CASCADE"
+        />
+    </changeSet>
+
+    <!-- Jira Could Job Tables -->
+
+    <changeSet author="gavink" id="create-jira-cloud-job-details-table">
+        <createTable schemaName="alert" tableName="jira_cloud_job_details">
+            <column name="job_id" type="UUID">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="add_comments" type="BOOLEAN" defaultValueBoolean="false">
+                <constraints nullable="false"/>
+            </column>
+            <column name="issue_creator_email" type="VARCHAR">
+                <constraints nullable="true"/>
+            </column>
+            <column name="project_name_or_key" type="VARCHAR">
+                <constraints nullable="false"/>
+            </column>
+            <column name="issue_type" type="VARCHAR">
+                <constraints nullable="false"/>
+            </column>
+            <column name="resolve_transition" type="VARCHAR">
+                <constraints nullable="true"/>
+            </column>
+            <column name="reopen_transition" type="VARCHAR">
+                <constraints nullable="true"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet author="gavink" id="create-jira-cloud-job-details-table-fk">
+        <addForeignKeyConstraint
+                constraintName="jira_cloud_job_details_job_id_fk"
+                baseTableSchemaName="alert"
+                baseTableName="jira_cloud_job_details"
+                baseColumnNames="job_id"
+                referencedTableSchemaName="alert"
+                referencedTableName="distribution_jobs"
+                referencedColumnNames="job_id"
+                onDelete="CASCADE"
+                onUpdate="CASCADE"
+        />
+    </changeSet>
+
+    <!-- Jira Server Job Tables -->
+
+    <changeSet author="gavink" id="create-jira-server-job-details-table">
+        <createTable schemaName="alert" tableName="jira_server_job_details">
+            <column name="job_id" type="UUID">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="add_comments" type="BOOLEAN" defaultValueBoolean="false">
+                <constraints nullable="false"/>
+            </column>
+            <column name="issue_creator_username" type="VARCHAR">
+                <constraints nullable="true"/>
+            </column>
+            <column name="project_name_or_key" type="VARCHAR">
```

Comment:
Can/Should we decide to use either name of key here instead of either?

Reply:
Good question. Either is valid for Jira, so we're sticking with the or-name.